mods.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 mkdir -p plugins
 cd plugins || exit
```

Comment:
Why is the exit command piped from a change directory command? Was that just for debugging and you forgot to remove it, or what?

Reply:
Its like if the cd fails then exit the script

Reply:
I'm linting it with this https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck
And heres its explaination https://www.shellcheck.net/wiki/SC2164

Reply:
Cool